tests/Headers.hs

```diff
@@ -310,7 +310,7 @@ multipleMailboxes =
 
 -- | Generate headers
 genFieldItem :: Gen B.ByteString
-genFieldItem = resize 55 . B.pack <$> listOf1 (suchThat arbitrary isFtext)
+genFieldItem = resize 55 $ listOf1 (suchThat arbitrary isFtext) >>= pure . B.pack
```

Comment:
Thanks, nice pickup.  But we can still appease hlint by
```haskell
resize 55 $ B.pack <$> listOf1 (suchThat arbitrary isFtext)
```

Reply:
Aaah doh. I was banking on my editor. Sorry.